### PR TITLE
Add plugin "foreach"

### PR DESCRIPTION
**Pull request by ahmetb**
Runs a specified kubectl command across multiple contexts.

This plugin is similar to `mc` and `allctx`, but differentiates around concurrency controls, more sophisticated inclusion/exclusion patterns, output prefixing and confirmation prompts.

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/foreach.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      uri: https://github.com/ahmetb/kubectl-foreach/releases/download/v0.1.0/kubectl-foreach_v0.1.0_linux_arm64.tar.gz
+    sha256: c1d67b5d02122fe8514b4f768c829f5d3986661083bec8cf0c242d7c03448e10
+      bin: kubectl-foreach
+    - selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+      uri: https://github.com/ahmetb/kubectl-foreach/releases/download/v0.1.0/kubectl-foreach_v0.1.0_windows_amd64.tar.gz
+    sha256: 2384493f126f10cf3fa07fc252058155d64b975fcc100c20ca16caa0e1898ead
+      bin: kubectl-foreach.exe
```
Comment (chriskim06) [inline plugins/foreach.yaml:49]:
```suggestion
  platforms:
  - selector:
      matchLabels:
        os: darwin
        arch: amd64
    uri: https://github.com/ahmetb/kubectl-foreach/releases/download/v0.1.0/kubectl-foreach_v0.1.0_darwin_amd64.tar.gz
    sha256: a1c572f64dadfca7088421adad8b203f520fbe251ab08d61501d4629665bad86
    bin: kubectl-foreach
  - selector:
      matchLabels:
        os: darwin
        arch: arm64
    uri: https://github.com/ahmetb/kubectl-foreach/releases/download/v0.1.0/kubectl-foreach_v0.1.0_darwin_arm64.tar.gz
    sha256: 6d2b1d715ba81d1a0c40a9f9e93288175f67e4d14252dec33df928a62459d27e
    bin: kubectl-foreach
  - selector:
      matchLabels:
        os: linux
        arch: amd64
    uri: https://github.com/ahmetb/kubectl-foreach/releases/download/v0.1.0/kubectl-foreach_v0.1.0_linux_amd64.tar.gz
    sha256: ed93b7d02b59927f18fea23f9c0d07321adaf15af5afee52ec801846d0cf722e
    bin: kubectl-foreach
  - selector:
      matchLabels:
        os: linux
        arch: arm64
    uri: https://github.com/ahmetb/kubectl-foreach/releases/download/v0.1.0/kubectl-foreach_v0.1.0_linux_arm64.tar.gz
    sha256: c1d67b5d02122fe8514b4f768c829f5d3986661083bec8cf0c242d7c03448e10
    bin: kubectl-foreach
  - selector:
      matchLabels:
        os: windows
        arch: amd64
    uri: https://github.com/ahmetb/kubectl-foreach/releases/download/v0.1.0/kubectl-foreach_v0.1.0_windows_amd64.tar.gz
    sha256: 2384493f126f10cf3fa07fc252058155d64b975fcc100c20ca16caa0e1898ead
    bin: kubectl-foreach.exe
```

shas are indented incorrectly

File: plugins/foreach.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  name: foreach
+spec:
+  version: v0.1.0
+  homepage: https://github.com/ahmetb/kubectl-foreach
+  shortDescription: Run kubectl commands against some/all contexts in parallel
+  description: |
+    Lets you run the same kubectl command against multiple contexts
+    simultaneously and prints their output, prefixed by context name.
+    You can choose or exclude contexts with exact name match or regular
+    expression patterns.
```
Comment (chriskim06) [inline plugins/foreach.yaml:13]:
```suggestion
    Run the same kubectl command against multiple contexts
    simultaneously and print their output, prefixed by context name.
    Choose or exclude contexts with exact name match or regular
    expressions.
```
minor nit but something written in the imperative sounds better imo, wdyt?

**Comment by ahmetb** [comment]
@chriskim06 thanks for review, incorporated suggested changes.

**Comment by chriskim06** [comment]
/lgtm

thanks! and nice plugin 👍

**Pull request merged**